src/hotspot/share/gc/shared/space.inline.hpp

```diff
@@ -125,14 +127,18 @@ class DeadSpacer : StackObj {
       log_develop_trace(gc, compaction)("Inserting object to dead space: " PTR_FORMAT ", " PTR_FORMAT ", " SIZE_FORMAT "b",
           p2i(dead_start), p2i(dead_end), dead_length * HeapWordSize);
 
-      _dead_space += dead_length;
       return true;
     } else {
       _active = false;
       return false;
     }
   }
 
+private:
+  size_t get_dead_space() {
+    return (_max_deadspace_words - _allowed_deadspace_words) * HeapWordSize;
+  }
```

Comment:
Hotspot does not use a "get_" prefix for getters. Also not sure why this needs to be private (and the friend class), I would prefer this instead of the friending. Retrieving the actual amount of dead space from a class that calculates it does not  seem something that needs hiding.

Reply:
The visibility for `_dead_space` was changed based on this comment: https://github.com/openjdk/jdk/pull/2579/files/f69541864e093bc5b250bf625ec75983764ba2bf#r585771280 by @shipilev

Reply:
Also, I see a bunch of methods named `get_*` in GC code alone. I have no problem renaming it to eg. `dead_space()` but it does not seem that this naming pattern is not used in hotspot.

Reply:
https://wiki.openjdk.java.net/display/HotSpot/StyleGuide

> Nearly all of the guidelines mentioned below have many counter-examples in the Hotspot code base. Finding a counterexample is not sufficient justification for new code to follow the counterexample as a precedent, since readers of your code will rightfully expect your code to follow the greater bulk of precedents documented here. For more on counterexamples, see the section at the bottom of this page.
> 
>    When changing pre-existing code, it is reasonable to adjust it to match these conventions. Exception: If the pre-existing code clearly conforms locally to its own peculiar conventions, it is not worth reformatting the whole thing.

and

>  Getter accessor names are noun phrases, with no "get_" noise word. Boolean getters can also begin with "is_" or "has_".

Unless there is a good reason, please keep to few rules the official style guide for new code.

Reply:
:+1: